src/pages/main/index.tsx

```diff
@@ -1,16 +1,48 @@
-import React from "react";
+import React, { useEffect } from "react";
+import { Wrapper } from "@googlemaps/react-wrapper";
 
-import { toggleMenu } from "@/state/features";
+import UserLocation from "@/assets/icons/UserLocation.svg";
 import { useAppDispatch, useAppSelector } from "@/state/app/hooks";
-import { Navbar, ContactModal } from "@components/common";
-import { Welcome } from "@components/main/welcome";
+import { Map, Marker, Welcome, Navbar, ContactModal } from "@/components";
+import { toggleMenu } from "@/state/features";
+import { getNavigationPermissions } from "@/services";
 
 import styles from "./Main.module.css";
 
 const Main = () => {
   const dispatch = useAppDispatch();
   const { menuOpen } = useAppSelector((state) => state.menu);
 
+  const [clicks, setClicks] = React.useState<google.maps.LatLng[]>([]);
```

Comment:
I know the docs is like this but I don't like too much this state as `clicks`. It's not clear for me. WDYT

Reply:
I think that state will be useful for the creation of new targets afterwards, so I can rename it `[targets, setTargets]` WDYT?

Reply:
I'm not sure if there are the same. Because targets are places that you have created and saved but this state is only when you click anywhere right? In this project I don't think that you need to save all the "clicks" only the last one. It's just and idea, we could talk about this if it's not clear.